tests/conftest.py

```diff
@@ -350,11 +350,112 @@ def initial_data(logged_on_user):
             'stream_weekly_traffic': 0
         }],
         'unread_msgs': {
-            'pms': [],
-            'count': 0,
+            'pms': [{
```

Comment:
I like the addition of this extra data to the tests - but this is a lot. Is this real data you've captured? Can we trim it to base data and then parametrize to look at some cases? This looks like a lot, and while having real data is good, it makes it more difficult to understand?